tensorflow/contrib/repeat/BUILD

```diff
@@ -0,0 +1,80 @@
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+load("//tensorflow:tensorflow.bzl", "tf_custom_op_library")
+load("//tensorflow:tensorflow.bzl", "tf_gen_op_libs")
+load("//tensorflow:tensorflow.bzl", "tf_gen_op_wrapper_py")
+load("//tensorflow:tensorflow.bzl", "tf_custom_op_py_library")
```

Comment:
Replacing `tf_custom_op_py_library` with `py_library` worked for me.  Needed to also replace `dso` with `data` in the `py_library` stanza (just pattern-matching from other material in contrib).